tests/testthat/test_correctness_bgnbd.R

```diff
@@ -15,6 +15,37 @@ test_that("cdnow nocov correct coefs and SE", {
   expect_equal(coef(e.bgnbd.cdnow.nocov), c(r = 0.2425945, alpha = 4.4136019, a = 0.7929199, b = 2.4258881))
 })
 
+# Is used at various places
+expect_silent(clv.cdnow <- clvdata(data.transactions = cdnow, date.format = "ymd", time.unit = "W",
+                                   estimation.split = 38))
+
+test_that("Same results as BTYD", {
+  # Fitting
+  # From ?BTYD::bgnbd.cbs.LL()
+  data(cdnowSummary, package = "BTYD")
+  expect_silent(cal.cbs <- cdnowSummary$cbs)
+  expect_silent(startingparams <- c(1, 3, 1, 3))
+  expect_silent(est.params <- BTYD::bgnbd.EstimateParameters(cal.cbs, startingparams))
+
+  # CLVTools
+  expect_silent(clv.cdnow <- clvdata(data.transactions = cdnow, date.format = "ymd", time.unit = "w", estimation.split = "1997-09-30"))
+  expect_silent(p.cdnow <- bgnbd(clv.data=clv.cdnow, start.params.model = c(r=1, alpha = 3, a = 1, b = 3), verbose=FALSE))
```

Comment:
start param b = beta?

Reply:
Actually, there is no beta parameter in the BG/NBD model